1-js/05-data-types/02-number/article.md

```diff
@@ -125,7 +125,7 @@ There are following built-in functions for rounding:
 : Rounds to the nearest integer: `3.1` becomes `3`, `3.6` becomes `4` and `-1.1` becomes `-1`.
 
 `Math.trunc` (not supported by Internet Explorer)
-: Removes the decimal part: `3.1` becomes `3`, `-1.1` becomes `-1`.
+: Removes anything after the decimal without rounding: `3.1` becomes `3`, `-1.1` becomes `-1`.
```

Comment:
"Removes anything after the decimal" - are you sure about it? Maybe you mean "after the decimal point"? 

Are you sure the original text is wrong?

Reply:
I had to look this one up, but I have a feeling @iliakan already knew :) The word '**decimal**' refers to **either** any numeral in the decimal system (aka base 10 system) **OR** any digit following the **decimal separator**. '**Decimal point**' refers to the separator itself. So, yea @gwooly should have said "anything after the decimal point'...